filebeat/input/stdin/input.go

```diff
@@ -48,6 +54,10 @@ type Input struct {
 // NewInput creates a new stdin input
 // This input contains one harvester which is reading from stdin
 func NewInput(cfg *common.Config, outlet channel.Connector, context input.Context) (input.Input, error) {
+	deprecatedNotificationOnce.Do(func() {
+		cfgwarn.Deprecate("", "Stdin input is deprecated.")
```

Comment:
Should we add this to the changelog too?

Reply:
I accidentally added this file. I wanted to create a separate PR, so I only have to add a single line to the changelog.